### PR TITLE
Implement configuration parser

### PR DESCRIPTION
**Pull request by kreemer**
This PR adds the ability to configure the bundle with a seperate configuration file.

Additionally I reworked the info endpoint so that the separate info collector could be en- or disabled.

### CODE REVIEW

**Comment by kreemer** [comment]
@akondas could you take a look?

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            ->canBeDisabled()
+            ->children()
+            ->integerNode('threshold')->defaultValue(50 * 1024 * 1024)->end()
+            ->scalarNode('path')->defaultValue('%kernel.project_dir%')->end()
+            ->end()
+            ->end()
+            ->end()
+            ->end()
+            ->end()
+            ->end()
```
Comment (akondas) [inline src/DependencyInjection/Configuration.php:36]: can we add some indentation here?

File: src/Resources/config/routing.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# Old configuration file with yml extension
```
Comment (akondas) [inline src/Resources/config/routing.yml:1]: should we remove this file?

**Review by akondas — APPROVED** [review]
LGTM

File: src/DependencyInjection/Configuration.php (hunk 1 above)
Comment (kreemer) [reply src/DependencyInjection/Configuration.php:36]: The next PR will add indentation

File: src/Resources/config/routing.yml (hunk 2 above)
Comment (akondas) [reply src/Resources/config/routing.yml:1]: make sense :+1:

**Comment by akondas** [comment]
Thanks :beers:

**Pull request merged**